api/v1alpha1/groupversion_info.go

```diff
@@ -12,16 +12,16 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{
-		Group:   "kargo.akuity.io",
-		Version: "v1alpha1",
+	GroupVersion	= schema.GroupVersion{
+		Group:		"kargo.akuity.io",
+		Version:	"v1alpha1",
```

Comment:
Where did all this extra space come from? This doesn't look gofmt'ed.

Same question in a lot of other places.

Reply:
Resolved in https://github.com/akuity/kargo/pull/1515#discussion_r1500519314